app/lib/ddp.js

```diff
@@ -39,6 +39,11 @@ class EventEmitter {
 			}
 		}
 	}
+	removeAllForEvent(event) {
```

Comment:
We fixed some bugs at #379.
`removeAllForEvent` doesn't seem to be necessary anymore, since every subscribe closes its listener.

Reply:
Hmm, I made this patch from a version of ddp.js that I have in typescript (made for a specific React Native project). In my case, I needed it so that I could clear all events that were added while the stream 'stream-room-messages' was open

Reply:
Ok. Sounds like it's a different behaviour than ours.
In our case, we would clear events one by one.
For us, `removeAllForEvent` could cause unpredictable behaviour in future.